conf.py

```diff
@@ -676,6 +676,11 @@
 #    ".js": [filters.closure_compiler],
 #    ".jpg": ["jpegoptim --strip-all -m75 -v %s"],
 # }
+from nikola import filters
+
+FILTERS = {
+    ".html": [filters.add_header_permalinks],
+}
```

Comment:
This introduced a visual regression:

<details>

After this PR:
<img width="468" alt="Screenshot 2024-03-22 at 00 40 21" src="https://github.com/hpyproject/hpyproject.org/assets/13780613/87cca2b8-eaa7-4a56-b4b0-e135d23b10d9">

Before this PR:
<img width="450" alt="Screenshot 2024-03-22 at 00 40 43" src="https://github.com/hpyproject/hpyproject.org/assets/13780613/72fc432e-c71a-4c64-bd05-64c03ebf6c4a">


</details>

Search works fine without the permalinks; consider removing them for less visual clutter.

Reply:
Perhaps we can just style them better? Being able to link to subheadings on pages is nice.

Reply:
Sure, that would be a nice solution. I'll create an issue for it, so it's not forgotten :)